src/components/SpaceProposalVotesModal.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
+import { ref, toRefs, watch } from 'vue';
+import { useScroll } from '@vueuse/core';
+import { useProposalVotesList } from '@/composables';
+import {
+  ExtendedSpace,
+  Proposal,
+  Vote,
+  SpaceStrategy
+} from '@/helpers/interfaces';
+
+const props = defineProps<{
+  space: ExtendedSpace;
+  proposal: Proposal;
+  strategies: SpaceStrategy[];
+  userVote: Vote | null;
+  open: boolean;
+}>();
+
+defineEmits(['close']);
+
+const {
+  votes,
+  loadVotes,
+  sortedVotes,
+  loadMore,
+  loadingMore,
+  loadMoreVotes,
+  profiles
+} = useProposalVotesList(props.proposal, props.userVote);
+
+const votesScrollWrapper = ref<HTMLElement | null>(null);
+const votesQuery = ref('');
+
+const { arrivedState } = useScroll(votesScrollWrapper, {
+  throttle: 500,
+  offset: {
+    bottom: 300
+  }
+});
+
+const { bottom } = toRefs(arrivedState);
+
+watch(
+  () => bottom.value,
+  to => {
+    if (props.open && to) {
+      loadMore(loadMoreVotes);
+    }
+  },
+  {
+    immediate: false
+  }
+);
+
+watch(
+  () => props.open,
+  to => {
+    if (to) {
+      loadVotes(20);
+    } else {
+      votes.value = [];
+    }
+  }
+);
+</script>
+
+<template>
+  <BaseModal
+    :open="open"
+    max-height="608px"
+    max-width="608px"
```

Comment:
To make it would with the our default width we can only show the avatar imagine without the address inside the modal.

Reply:
I would still recommend doing this as it all seems a bit tight when choices are longer. See screenshot:
<img width="494" alt="image" src="https://user-images.githubusercontent.com/51686767/226818072-ca5c55f3-64ec-4777-a431-35bc1daaf7f9.png">

Reply:
Still don't know why we need default width for all our modals, we have different content inside them...

Reply:
![image](https://user-images.githubusercontent.com/97898172/226884720-4653d82a-644d-4bb7-8b3a-73f57d519150.png)
it is so narrow that even the address barely fits in